test/integration/operator/garden/garden/garden_test.go

```diff
@@ -409,8 +410,8 @@ spec:
 			g.Expect(testClient.List(ctx, crdList)).To(Succeed())
 			return test.ObjectNames(crdList)
 		}).WithTimeout(kubernetesutils.WaitTimeout).Should(ContainElements(
-			"etcds.druid.gardener.cloud",
-			"etcdcopybackupstasks.druid.gardener.cloud",
+			druidcorecrds.ResourceNameEtcd,
+			druidcorecrds.ResourceNameEtcdCopyBackupsTask,
```

Comment:
Revert

Reply:
addressed in https://github.com/gardener/gardener/pull/11545/commits/cdf2536229ecc8936f22fd06d7c857081cb7932b